src/views/Register/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Form, Input, Button, Row, Col, Icon, message } from 'antd'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom';
+import { login } from '../../actions/auth'
+
+const FormItem = Form.Item
+
+import './index.less'
+
+const propTypes = {
+  user: PropTypes.object,
+  loggingIn: PropTypes.bool,
+  loginErrors: PropTypes.string
+};
+
+class Register extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      loading: false
+    }
+  }
+
+  handleSubmit (e) {
+    e.preventDefault();
+    this.setState({
+      loading: true
+    });
+    const data = this.props.form.getFieldsValue()
+    if (data.user ===  undefined || data.password === undefined) {
+      this.setState({
+        loading: false
+      });
+      message.error("Incorrect user or password");
+    } else {
+      this.setState({
+        loading: false
+      });
+      message.success("Welcome " + data.user)
```

Comment:
After registering successfully, I think it's better to redirect to `/login`, Beacuse user has not login, if redirect to `/home`, then page will still redirect to `/login` because of not logining.